action.yaml

```diff
@@ -18,5 +18,17 @@ inputs:
     required: false
     default: ""
 runs:
-  using: 'node20'
-  main: 'dist/index.mjs'
+  using: "composite"
+  steps:
+    - name: Setup Bun
+      uses: oven-sh/setup-bun@v1
+      with:
+        bun-version: 1.x.x
+
+    - name: Install dependencies
+      run: bun i
+      shell: bash
+
+    - name: Setup Biome
+      run: bun .
+      shell: bash
```

Comment:
The setup script must be bundled as a self-contained executable, because we cannot install its dependencies. Also, this executable must be compatible with the three major operating systems: macOS, linux, windows.

Reply:
noted!